src/test/java/org/springframework/data/couchbase/domain/CapellaConnectSample.java

```diff
@@ -90,10 +90,11 @@ public static void main(String... args) {
 		}
 
 		ClusterEnvironment env = ClusterEnvironment.builder()
-				.securityConfig(SecurityConfig.enableTls(true)/*.trustManagerFactory(InsecureTrustManagerFactory.INSTANCE)*/)
+				//.securityConfig(SecurityConfig.enableTls(true)/*.trustManagerFactory(InsecureTrustManagerFactory.INSTANCE)*/)
```

Comment:
Do we want this commenting left in the example?

Reply:
Oops.  This file shouldn't be in this branch.